src/configure.py

```
@@ -1031,7 +1031,7 @@ def validation( paths, depends, constraints, **kwargs ):
             if kwargs[opt] != opt_val: continue         # not the value to be checked
             for check_opt, check_val in check.items():
                 if type(check_val) != type([]): check_val = [check_val]   # transform to list
-                if kwargs[check_opt] in check_val: continue     # satisify the validation
+                if kwargs[check_opt] in check_val: continue     # satisfy the validation
```

Ruff (0.15.2)

[error] Multiple statements on one line (colon)

(E701)